Feat/master

leesumin7766 opened the pull request:
## 🔗 Issue 번호
- close #32

## 🛠 작업 내역
-마스터 계정으로 매니저 관리 기능 추가

## 🔄 변경 사항
-

## ✨ 새로운 기능
-

## 📦 작업 유형
- [ ] 신규 기능 추가
- [ ] 버그 수정
- [ ] 리팩토링
- [ ] 문서 업데이트

## ✅ 체크리스트
- [ ] Merge 대상 branch가 올바른가?
- [ ] 약속된 컨벤션 (on code, commit, issue...) 을 준수하는가?
- [ ] PR과 관련없는 변경사항이 없는가?

leesumin7766 committed:
add: 매니저 관리 기능 업데이트

leesumin7766 committed:
add: 매니저 관리 기능 업데이트

leesumin7766 committed:
add: 매니저 관리 기능 업데이트

leesumin7766 committed:
add: 매니저 관리 기능 업데이트

leesumin7766 committed:
Merge branch 'dev' of https://github.com/dding-dong/dingdong into feat/master

leesumin7766 committed:
add: 매니저 관리 기능 업데이트

leesumin7766 committed:
add: 매니저 관리 기능 업데이트

pleasebelieveme reviewed (commented):
일단 코멘트 적어봅니다!ㅎㅎ

leesumin7766 committed:
fix : User_Id -> Id
V1 추가

leesumin7766 committed:
fix : User_Id -> Id
V1 추가

gashine20 reviewed (commented):
수민님 코드 잘봤습니다~~! 깔끔해서 읽기 편했습니다 👍
멘트 남겨놨습니다!!

leesumin7766 committed:
feat :order API 생성
결제 로직하고 맞추기 전,
예외 처리 전

leesumin7766 committed:
A
Merge branch 'dev' into feat/order

leesumin7766 committed:
feat : order API 생성
결제 로직하고 맞추기 전,
예외 처리 전

pleasebelieveme committed:
feat: #64 리드미 초안 작성

leesumin7766 committed:
Merge branch 'dev' of https://github.com/dding-dong/dingdong into feat/master

leesumin7766 committed:
fix : # 32
master -> admin으로 변경
Setter 제외 비즈니스 매서드로 변경
@PathVariable 제외
@Get으로 매니저 목록 조회
@PATCH로 상태 변경

leesumin7766 committed:
fix : # 32
master -> admin으로 변경
Setter 제외 비즈니스 매서드로 변경
@PathVariable 제외
@Get으로 매니저 목록 조회
@PATCH로 상태 변경

leesumin7766 committed:
fix : # 32
master -> admin으로 변경
Setter 제외 비즈니스 매서드로 변경
@PathVariable 제외
@Get으로 매니저 목록 조회
@PATCH로 상태 변경

# Conflicts:
#	src/main/java/com/sparta/dingdong/common/exception/CommonErrorCode.java

leesumin7766 committed:
fix : # 32
master -> admin으로 변경
Setter 제외 비즈니스 매서드로 변경
@PathVariable 제외
@Get으로 매니저 목록 조회
@PATCH로 상태 변경

leesumin7766 committed:
fix : PENDING 취소 시 단순 상태 변환
REQUESTED 취소 시 결제 취소 호출
양방향 순환 의존성 문제로 Lazy 추가

pleasebelieveme committed:
feat: #64 리드미 1번째 수정

pleasebelieveme committed:
feat: #64 리드미 2번째 수정

pleasebelieveme committed:
Merge pull request #66 from dding-dong/feat/readme

feat: 리드미 초안 작성

malaheaven committed:
refactor: 공통 리뷰 전체 조회 문구 변경

malaheaven committed:
refactor: 스토어에서 값 가져오는 것으로 변경

leesumin7766 committed:
feat : V1 브렌치 생성

chachohee committed:
Merge pull request #69 from dding-dong/refactor/reviews

Refactor/reviews

chachohee committed:
feat: #70 AI 프롬프트 생성 로직 분리

leesumin7766 committed:
fix : user의 dongId가 Store의 deliveryAreaIds 안에 포함되지 않으면 배달불가지역 로그 추가

chachohee committed:
Merge pull request #71 from dding-dong/feat/ai-promptbuilder

feat: #70 AI 프롬프트 생성 로직 분리

leesumin7766 committed:
Merge branch 'dev' of https://github.com/dding-dong/dingdong into feat/orderV1

malaheaven committed:
feat: GlobalException MethodArgumentNotValidException handle 할 수 있도록 개발

malaheaven committed:
refactor: payment @Valid 사용할 수 있도록 수정

malaheaven committed:
refactor: review @Validated를 @Valid로 수정

malaheaven committed:
Merge pull request #73 from dding-dong/refactor/review-v2

Refactor/review v2

pleasebelieveme reviewed (dismissed):
LGTM!

leesumin7766 committed:
fix : 순환참조 해결
user의 dong, store의 deliveryAreaId 맞춰서 주소기반

leesumin7766 committed:
fix : serviceimpl에서 repo기반 참조에서 service기반 참조로 변경

leesumin7766 committed:
fix : Controller 분기

leesumin7766 committed:
Merge branch 'dev' of https://github.com/dding-dong/dingdong into feat/orderV1

leesumin7766 committed:
fix : 분기 후 로직 완성, admin 상태 변경 이력 조회 x

leesumin7766 committed:
feat : order exception 8000번대 사용

pleasebelieveme committed:
fix: #68 jwt 버전관리 방식으로 수정

leesumin7766 committed:
feat : AdminOrder history 추가

leesumin7766 committed:
feat : PaymentPageServiceImpl
결제시점 setRequestAt추가

pleasebelieveme committed:
Merge pull request #75 from dding-dong/fix/auth-tokenversion

fix: #68 jwt 버전관리 방식으로 수정

leesumin7766 committed:
feat : 최소 주문 금액 추가

chachohee committed:
feat: #76 application.yml prod, dev 분리

malaheaven committed:
Merge pull request #77 from dding-dong/feat/yml

feat: #76 application.yml prod, dev 분리

leesumin7766 committed:
fix : 히스토리 id값

malaheaven committed:
feat: review createReview TestCode 개발

chachohee committed:
feat: #78 배달가능지역으로 가게 검색 추가

pleasebelieveme committed:
Update README.md

chachohee committed:
Merge pull request #79 from dding-dong/feat/store-search-with-dong

feat: #78 배달가능지역으로 가게 검색 추가

chachohee committed:
Merge pull request #80 from dding-dong/pleasebelieveme-patch-1

dev 브런치 리드미 수정

malaheaven committed:
Merge pull request #82 from dding-dong/feat/review-testcode

feat: review createReview TestCode 개발

leesumin7766 committed:
fix : db파일 삭제

leesumin7766 committed:
feat : 병합

leesumin7766 committed:
Merge pull request #72 from dding-dong/feat/orderV1

Feat/order v1

malaheaven committed:
refactor: order 줄맞춤

malaheaven committed:
Merge pull request #83 from dding-dong/refactor/order

refactor: order 줄맞춤

pleasebelieveme committed:
feat: 매니저 전체 조회, 매니저 상태변경 메서드 구현

pleasebelieveme committed:
feat: 매니저 전체 조회, 매니저 상태변경 메서드 구현

pleasebelieveme committed:
Merge remote-tracking branch 'origin/feat/master' into feat/master

# Conflicts:
#	src/main/java/com/sparta/dingdong/common/exception/CommonErrorCode.java
